README.MD

```diff
@@ -1,2 +1,7 @@
 ### GIT
-THis is the first commit from main branch
\ No newline at end of file
+THis is the first commit from main branch
+
+
+### feature-1
+* developer is doing some development
```

Comment:
do we really need this line

Reply:
yes we need for sure otherwise will give some errrors